app/src/main/res/layout/fragment_home.xml

```diff
@@ -126,6 +127,26 @@
             android:text="@string/emptyList"
             android:textSize="20sp"
             android:textStyle="bold" />
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/refresh_button"
+            android:layout_width="150dp"
+            android:layout_height="50dp"
+            android:layout_gravity="center"
+            android:layout_marginTop="30dp"
+            android:textSize="12sp"
+            app:backgroundTint="@color/exo_black_opacity_70"
+            android:text="@string/retry"/>
```

Comment:
Please remove these lines, the buttons look quite strange like that.

Either give them a style of `@style/Widget.Material3.Button.TextButton` or use `style="@style/Widget.Material3.Button.ElevatedButton` (I personally prefer elevated buttons for that purpose)